### DISCUSSION TITLE
Help with statcell

### DISCUSSION THREAD

**Discussion by jadball** [neperfepx/neper#127] [Q&A] [answered]
Hi all!

I'm new to Neper, so this might have a very simple answer.
Running the tessellation module with the following command yields an error:
```
neper -T -n from_morpho -domain "cube(1000,1000,1000)" -morpho "graingrowth(75)" -oricrysym cubic -ori random -oridescriptor rotmat:passive -regularization 1 -o 1mm_ferrite_cube_75um_cluster -format tess -statcell id,coo,vol,diameq
```
The error is:
```
Info   : Writing statistics...
Info   :     [o] Writing file `1mm_ferrite_cube_75um_cluster.stcell'...
Error  : Expression `coo' could not be processed.
Aborted
```

I would like statistics on the centre-of-mass positions of the generated cells. Am I doing the wrong thing by specifying `coo` for `-statcell`?

**Comment by rquey** [comment, marked as answer]
Which version are you using? because it works here with v4.1.2.

**Reply by jadball** [reply]
I was using new documentation but an older software version. Fixed by updating - thanks!

**Reply by sushdash** [reply]
I have a small doubt here! Do these 1000 units mean 1 mm over here?

**Reply by rquey** [reply]
Again, it is a length, but it can be 1000 meters, millimeters, micrometers, etc.
